common/tasks/install_configure_MariaDB.yaml

```diff
@@ -20,36 +20,36 @@
   - (container is undefined) or (container == False)
   ignore_errors: yes
 
-- name: setup the MariaDB repository
-  copy:
-    src: etc_yum.repos.d_MariaDB.repo
-    dest: /etc/yum.repos.d/MariaDB.repo
-    owner: root
-    group: root
-    mode: 0644
+# if this installs mariadb 11 ssl will break
+- name: ensure mariadb installed
```

Comment:
check to see if we really need 10, and if so try to install specifically 10

Reply:
changed to specify 10 [b35803e](https://github.com/hep-gc/uvic-heprc-ansible-playbooks/pull/14/commits/b35803eeb1be51449ec53508ecefa143c67f96ad)